src/views/EditSidebar.vue

```diff
@@ -28,9 +28,27 @@
 		:title-editable="!isReadOnly && !isLoading"
 		:title-placeholder="$t('calendar', 'Event title')"
 		:subtitle="subTitle"
+		:empty="isLoading || isError"
 		@close="cancel"
 		@update:title="updateTitle">
-		<template v-slot:primary-actions>
+		<template v-if="isLoading">
+			<div class="app-sidebar__loading-indicator">
+				<div class="icon icon-loading app-sidebar-tab-loading-indicator__icon" />
+			</div>
+		</template>
+
+		<template v-else-if="isError">
+			<EmptyContent icon="icon-calendar-dark">
+				{{ $t('calendar', 'Event does not exist') }}
+				<template #desc>
+					{{ error }}
```

Comment:
Having both `Event does not exist` and `The event does not exist` is a bit doubled, no? I think the main title would be enough.

Reply:
Otherwise we can add a sentence explaining things like « It might have been deleted, or there was a typo in a link ».

Reply:
I changed the error message as requested, looks much better now :)